omegatrade-app/frontend/src/app/components/company/update-company/update-company.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="spinner" *ngIf="this.loader">
```

Comment:
`*ngIf="this.loader"`
this.loader will not work, this word should not be used anywhere in the html file

Reply:
this issue has been resolved.